<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreationDialog</class>
 <widget class="QDialog" name="CreationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Load Complex Features source</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="filenameRadio">
       <property name="text">
        <string>Input filename</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="filenameText"/>
     </item>
     <item>
      <widget class="QToolButton" name="browseButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="urlRadio">
       <property name="text">
        <string>Input URL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="urlText">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Output filename</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="outFilenameText"/>
     </item>
     <item>
      <widget class="QToolButton" name="browseOutButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QComboBox" name="mImportTypeCombo">
     <item>
      <property name="text">
       <string>Import as one layer</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="mStackedWidgets">
     <widget class="QWidget" name="mLayerImportPage">
      <property name="minimumSize">
       <size>
        <width>954</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Attribute mapping</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="addMappingBtn">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeMappingBtn">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="attributeTable">
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Attribute</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Type</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>XPath</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QCheckBox" name="geometryColumnCheck">
           <property name="text">
            <string>Geometry column XPath</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="geometryColumnEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="replaceLayerChck">
         <property name="text">
          <string>Replace current layer</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mRelationalImportPage">
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CreationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CreationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filenameRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>filenameText</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filenameRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>browseButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>733</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>urlRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>urlText</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>geometryColumnCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>geometryColumnEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>450</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>453</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mImportTypeCombo</sender>
   <signal>activated(int)</signal>
   <receiver>mStackedWidgets</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>485</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>485</x>
     <y>451</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
